### ISSUE TITLE
0.35.31-alpha broke gradle

### ISSUE THREAD

**Issue by novotl** [airbytehq/airbyte#10541]
## Environment
- **Airbyte version**:  0.35.31-alpha
- **OS Version / Instance**: Debian 10, Gradle 7.3.3
- **Deployment**: non applicable
- **Severity**: Very Low / Low / Medium / High / Critical
- **Step where error happened**: Compile

## Current Behavior
*Tell us what happens.*
- Running `SUB_BUILD=PLATFORM ./gradlew :airbyte-workers:copyGeneratedTar` indefinitely waits for database connection
```
2022-02-22 13:33:48 WARN i.a.d.Databases(createPostgresDatabaseWithRetryTimeout):66 - Waiting for database to become available...
2022-02-22 13:33:48 INFO i.a.d.Databases(createPostgresDatabaseWithRetryTimeout):67 - user: docker, pass: docker, conn: jdbc:postgresql://db:5432/airbyte
2022-02-22 13:33:48 INFO i.a.d.i.j.JobsDatabaseInstance(lambda$static$2):25 - Testing if jobs database is ready...
2022-02-22 13:33:48 INFO i.a.d.Databases(createPostgresDatabaseWithRetryTimeout):76 - Database is not ready yet. Please wait a moment, it might still be initializing...
```

I've added a debug log here https://github.com/airbytehq/airbyte/blob/5464b1c830f6ff06dc1bef70c94b389c4085f6bd/airbyte-db/lib/src/main/java/io/airbyte/db/Databases.java#L53-L57

that reveals `jdbcConnectionString == "jdbc:postgresql://db:5432/airbyte"`. The postgres is automatically created automatically in docker. 
Previous **working** version had `jdbcConnectionString == "jdbc:postgresql://localhost:49194/jooq_airbyte_configs?loggerLevel=OFF"`

I think this might be the culprit https://github.com/airbytehq/airbyte/pull/10371

## Expected Behavior
- passing `SUB_BUILD=PLATFORM ./gradlew :airbyte-workers:copyGeneratedTar`

## Steps to Reproduce
1. `SUB_BUILD=PLATFORM ./gradlew :airbyte-workers:copyGeneratedTar`

## Are you willing to submit a PR?

Remove this with your answer.

**Comment by marcosmarxm** [comment]
@tuliren could you check this issue?

**Comment by tuliren** [comment]
I have looked into this, but cannot reproduce it locally. I have tested the `:airbyte-workers:copyGeneratedTar` on both version `0.35.31-alpha` and the latest head.

In addition, the platform build passed on CI in both the current head commit as well as the [PR](https://github.com/airbytehq/airbyte/pull/10446) that created the `0.35.31-alpha` version.

The PR linked to this issue https://github.com/airbytehq/airbyte/pull/10371 did not change anything related to actual database connection. It only removed some unnecessary classes.

So I don't think this is a universal problem.
